Usage: main [OPTIONS] PYPI_NAME

Error: No git repository was found at .
